vividus-plugin-web-app/src/main/java/org/vividus/selenium/WebDriverType.java

```diff
@@ -105,16 +107,15 @@ void setDriverExecutablePath(Optional<String> driverExecutablePath)
                     WebDriverManager::iedriver);
         }
     },
-    CHROME(true, true, false, Set.of(ChromeOptions.CAPABILITY), BrowserType.CHROME, BrowserType.GOOGLECHROME)
+    CHROME(true, true, false, Set.of(ChromeOptions.CAPABILITY), Browser.CHROME)
     {
         @Override
         public WebDriver getWebDriver(DesiredCapabilities desiredCapabilities, WebDriverConfiguration configuration)
         {
-            ChromeOptions options = new ChromeOptions();
+            ChromeOptions options = new ChromeOptions().merge(desiredCapabilities);
             configuration.getBinaryPath().ifPresent(options::setBinary);
             options.addArguments(configuration.getCommandLineArguments());
             configuration.getExperimentalOptions().forEach(options::setExperimentalOption);
-            options.merge(desiredCapabilities);
```

Comment:
Wasn't order necessary here?

Reply:
no, it wasn't, AFAIk you can't set binary path, args, etc. via capabilities